src/main/webapp/style.css

```diff
@@ -82,13 +82,6 @@
   float: right;
 }
 
-#organization-list {
```

Comment:
organizations are being added to this list div. its for encapsulation and styling purposes. is there a way to refactor your code so you dont get rid of this? could you put the see more results button as the last element in the div?